docs/modules/operation/pages/quick-start/users.adoc

```diff
@@ -1,7 +1,7 @@
 
-= Step 1: Login and change admin password
+= Step 1: Log In and Change `Admin` Password
```

Comment:
I don't like how the ` formats in the title.
```suggestion
= Step 1: First Time Log In
```

Reply:
Good call, that is an annoying formatting thing.